streams/src/test/java/org/apache/kafka/streams/kstream/internals/KTableAggregateTest.java

```diff
@@ -85,33 +85,41 @@ public void testAggBasic() {
 
         driver.setUp(builder, stateDir, Serdes.String(), Serdes.String());
 
+        driver.setTime(10L);
```

Comment:
These will need updating if we merge https://github.com/apache/kafka/pull/6732 . Sorry :/

Reply:
Can we maybe merge 6732 this one?